k8s-manifests/database/pv.yml

```
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: PersistentVolume
+
+metadata:
+  name: mongo-pv
+  namespace: three-tier  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)